src/components/SearchList.js

```diff
@@ -1,17 +1,17 @@
-import React from 'react';
+// import React from 'react';
 
-function SearchList({ albums }) {
-  return (
-    <ul className="album-list">
-      {albums.map((album) => (
-        <li key={album._id} className="album-item">
-          <img src={album.image} alt={album.albumName} className="album-image" />
-          <p className="album-title">{album.albumName}</p>
-          <p className="album-artist">{album.artistName}</p>
-        </li>
-      ))}
-    </ul>
-  );
-}
+// function SearchList({ albums }) {
+//   return (
+//     <ul className="album-list">
+//       {albums.map((album) => (
+//         <li key={album._id} className="album-item">
+//           <img src={album.image} alt={album.albumName} className="album-image" />
+//           <p className="album-title">{album.albumName}</p>
+//           <p className="album-artist">{album.artistName}</p>
+//         </li>
+//       ))}
+//     </ul>
+//   );
+// }
```

Comment:
I don't think I need `SearchList` at this moment thats why its comment out

Reply:
got it; I think we can probably go ahead and remove the file in this case :+1: